code/modules/research/fabricators.dm

```diff
@@ -297,6 +297,7 @@
 			*/
 			being_built.forceMove(L) //Put the thing in the lockbox
 			L.name += " ([being_built.name])"
+			L.fits_max_w_class = 0 //You can't put things back into these boxes to prevent breaking arbitrary items
```

Comment:
If someone wanted to destroy the captain's laser; they could do so in any number of ways